### PR TITLE
header sticky

### PR DESCRIPTION
**Pull request by luchux**
Does: 
- sticky navbar. 
- sticky menu on SidebarCard component for mobile. 

DRAFT: 
- on mobile seems to fail with the simple CSS position: sticky attribute. 
debugging

### CODE REVIEW

File: ~ [hunk 1]
```diff
@@ -0,0 +1,480 @@
+import { BigNumber, ethers } from 'ethers';
```
Comment (rossneilson) [inline ~]: ?

Comment (luchux) [reply ~]: what is this? haha. let me see. wait, what?

File: src/components/Guilds/Header/index.tsx [hunk 2]
```diff
@@ -33,6 +41,7 @@ const Header = () => {
 
   return (
     <HeaderWrapper as="header">
+      <div></div>
```
Comment (mprasanjith) [inline src/components/Guilds/Header/index.tsx]: Curious if this is something we need?

**Review by mprasanjith — CHANGES_REQUESTED** [review]

File: src/components/Guilds/Header/index.tsx (hunk 2 above)
Comment (luchux) [reply src/components/Guilds/Header/index.tsx]: yeah, part of the workaround

**Review by rossneilson — DISMISSED** [review]
Please create a follow up issue for the mobile implementation

**Review by mprasanjith — APPROVED** [review]

**Pull request merged**